lib/ansible/module_utils/avi_ansible_utils.py

```diff
@@ -28,6 +28,19 @@
 #
 
```

Comment:
Might be worth putting a comment on the top of this module to state why it's named this way